src/pybind/mgr/volumes/fs/async_cloner.py

```diff
@@ -252,8 +252,17 @@ def handle_clone_failed(fs_client, volspec, volname, index, groupname, subvolnam
 
 def handle_clone_complete(fs_client, volspec, volname, index, groupname, subvolname, should_cancel):
     try:
-        with open_clone_sv_pair_in_vol(fs_client, volspec, volname, groupname,
-            subvolname) as (subvol0, subvol1, subvol2):
+        # XXX: Not using lock will result in race condition between cloner
```

Comment:
My bad, I had forgotten that we had decided to make lockless=True for new helper functions since it's been plenty months. Had I remembered, this mess could've been avoided.